CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp.bib

```diff
@@ -2372,3 +2370,238 @@ @article{1742-6596-664-8-082046
+     year = 2013,
+    month = dec,
+   adsurl = {http://adsabs.harvard.edu/abs/2013arXiv1312.6114K},
+  adsnote = {Provided by the SAO/NASA Astrophysics Data System}
+}
+@BOOK{NAP12980,
+  author    = "National Research Council",
+  editor    = "Samuel H. Fuller and Lynette I. Millett",
+  title     = "The Future of Computing Performance: Game Over or Next Level?",
+  isbn      = "978-0-309-15951-7",
+  doi       = "10.17226/12980",
+  abstract  = "The end of dramatic exponential growth in single-processor performance marks the end of the dominance of the single microprocessor in computing. The era of sequential computing must give way to a new era in which parallelism is at the forefront. Although important scientific and engineering challenges lie ahead, this is an opportune time for innovation in programming systems and computing architectures. We have already begun to see diversity in computer designs to optimize for such considerations as power and throughput. The next generation of discoveries is likely to require advances at both the hardware and software levels of computing systems.\n\nThere is no guarantee that we can make parallel computing as common and easy to use as yesterday's sequential single-processor computer systems, but unless we aggressively pursue efforts suggested by the recommendations in this book, it will be \"game over\" for growth in computing performance. If parallel programming and related software efforts fail to become widespread, the development of exciting new applications that drive the computer industry will stall; if such innovation stalls, many other parts of the economy will follow suit. \n\nThe Future of Computing Performance describes the factors that have led to the future limitations on growth for single processors that are based on complementary metal oxide semiconductor (CMOS) technology. It explores challenges inherent in parallel computing and architecture, including ever-increasing power consumption and the escalated requirements for heat dissipation. The book delineates a research, practice, and education agenda to help overcome these challenges. The Future of Computing Performance will guide researchers, manufacturers, and information technology professionals in the right direction for sustainable growth in computer performance, so that we may all enjoy the next level of benefits to society.",
+  url       = "https://www.nap.edu/catalog/12980/the-future-of-computing-performance-game-over-or-next-level",
+  year      = 2011,
+  publisher = "The National Academies Press",
+  address   = "Washington, DC"
+}
+
+@book{Cook:2012:CPD:2430671,
+ author = {Cook, Shane},
+ title = {CUDA Programming: A Developer's Guide to Parallel Computing with GPUs},
+ year = {2013},
+ edition = {1st},
+ publisher = {Morgan Kaufmann Publishers Inc.},
+ address = {San Francisco, CA, USA},
+} 
+% isbn = {9780124159334, 9780124159884},
+
+@Manual{dask_ref,
```

Comment:
Seems you have added references for things that are already present in the .bib file. There's been a Dask ref in there, from the supporting A&I WG paper. That's what I used in my updates to the A&I chapter, BTW.

Reply:
yes, I was somewhat  annoyed to find these after I did the work to find them (having discussed that it would be hard to find references...) I don't think it does any harm.